docs/api.md

```diff
@@ -459,14 +459,15 @@ This sets the account password and resets wrapKb to a new random value.
 
 The accountResetToken is single-use, and is consumed regardless of whether the request succeeds or fails.
 
-Optionally a user can request a new `sessionToken`.
+The caller can optionally request a new `sessionToken` and `keyFetchToken`.
 
 ### Request
 
 ___Parameters___
 
 * authPW - the PBKDF2/HKDF stretched password as a hex string
-* sessionToken - request sessionToken as a boolean
+* sessionToken - boolean, whether to generate a new sessionToken
```

Comment:
To support old clients, this field should be optional as well. I imagine it'd default to false.